### PR TITLE
update README, add CI

### PR DESCRIPTION
**Pull request by cyjseagull**
1. update README
2. add init_env.sh to install python-3.7.3
3. fix mispelling
4.  add ci

pls refer to:
https://github.com/cyjseagull/python-sdk

to read documentation, pls refer to:
https://cyjseagull-doc.readthedocs.io/zh_CN/release-2.0-2.0.0/docs/sdk/python_sdk/index.html

### CODE REVIEW

File: contracts/HelloWorld.bin [hunk 1]
```diff
@@ -0,0 +1 @@
...016020809104026020016040519081016040528093929190818152602001838380828437820191505050505050919291929050505061014a565b005b3480156100c657600080fd5b506100cf610164565b6040518080602001828103825283818151815260200191508051906020019080838360005b8381101561010f5780820151818401526020810190506100f4565b50505050905090810190601f16801561013c5780820380516001836020036101000a031916815260200191505b509250505060405180910390f35b8060009080519060200190610160929190610206565b5050565b606060008054600181600116156101000203166002900480601f0160208091040260200160405190810160405280929190818152602001828054600181600116156101000203166002900480156101fc5780601f106101d1576101008083540402835291602001916101fc565b820191906000526020600020905b8154815290600101906020018083116101df57829003601f168201915b5050505050905090565b828054600181600116156101000203166002900490600052602060002090601f016020900481019282601f1061024757805160ff1916838001178555610275565b82800160010185558215610275579182015b82811115610274578251825591602001919060010190610259565b5b5090506102829190610286565b5090565b6102a891905b808211156102a457600081600090555060010161028c565b5090565b905600a165627a7a72305820809ef12fcb7856c0e384cf1978f40931b23be16d1f0a583b8f49fd89d24aeb3e0029
```
Comment (bxq2011hust) [inline contracts/HelloWorld.bin]: this should be deleted.

File: contracts/HelloWorld.abi [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"constant":false,"inputs":[{"name":"n","type":"string"}],"name":"set","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"get","outputs":[{"name":"","type":"string"}],"payable":false,"stateMutability":"view","type":"function"},{"inputs":[],"payable":false,"stateMutability":"nonpayable","type":"constructor"}]
```
Comment (bxq2011hust) [inline contracts/HelloWorld.abi]: this should be deleted.

File: contracts/HelloWorld.bin (hunk 1 above)
Comment (cyjseagull) [reply contracts/HelloWorld.bin]: This is used for documentation, pls refer to: 
https://fisco-bcos-documentation.readthedocs.io/zh_CN/feature-python-sdk/docs/sdk/python_sdk/console.html

File: contracts/HelloWorld.abi (hunk 2 above)
Comment (cyjseagull) [reply contracts/HelloWorld.abi]: This is used for documentation, pls refer to:
https://fisco-bcos-documentation.readthedocs.io/zh_CN/feature-python-sdk/docs/sdk/python_sdk/console.html

**Pull request merged**